### DISCUSSION TITLE
Add some themes

### DISCUSSION THREAD

**Discussion by OrO-c** [Taewan-P/gpt_mobile#100] [Ideas]
Add some themes with different main colors (e.g. yellow, purple...)

**Comment by Taewan-P** [comment]
This is available through Dynamic Theme starting from Android 12. Just enable Dynamic Theme in settings and setting your system theme will change the app color as well.

**Reply by OrO-c** [reply]
What about phones below Android 12

**Reply by Taewan-P** [reply]
For that, we don't support theming at the moment.

**Comment by Taewan-P** [comment]
From version 0.6.0, I decided to drop support below Android 12. This is due to the new AICore SDK minimum SDK [requirement](https://developer.android.com/ai/gemini-nano/experimental#configure-gradle)..
